assets/javascripts/discourse/templates/components/wizard-custom-field.hbs

```diff
@@ -1,227 +1,227 @@
+      <label>{{i18n "admin.wizard.field.file_types"}}</label>
+    </div>
+    
+    <div class="setting-value">
+      {{input value=field.file_types class="medium"}}
+    </div>
+  </div>
+{{/if}}
+
+{{#if showLimit}}
+  <div class="setting">
+    <div class="setting-label">
+      <label>{{i18n "admin.wizard.field.limit"}}</label>
+    </div>
+    
+    <div class="setting-value">
+      {{input type="number" value=field.limit class="small"}}
+    </div>
+  </div>
+{{/if}}
+
+{{#if isDateTime}}
+  <div class="setting">
+    <div class="setting-label">
+      <label>{{html-safe (i18n "admin.wizard.field.date_time_format.label")}}</label>
+    </div>
+    
+    <div class="setting-value">
+      {{input value=field.format class="medium"}}
+      <label>{{html-safe (i18n "admin.wizard.field.date_time_format.instructions")}}</label>
+    </div>
+  </div>
+{{/if}}
+
+{{#if showPrefill}}
+  <div class="setting full field-mapper-setting">
+    <div class="setting-label">
+      <label>{{i18n "admin.wizard.field.prefill"}}</label>
+    </div>
+    
+    <div class="setting-value">
+      {{wizard-mapper
+        inputs=field.prefill
+        property="prefill"
+        onUpdate=(action "mappedFieldUpdated")
+        options=prefillOptions}}
+    </div>
+  </div>
+{{/if}}
+
+{{#if showContent}}
+  <div class="setting full field-mapper-setting">
+    <div class="setting-label">
+      <label>{{i18n "admin.wizard.field.content"}}</label>
+    </div>
+    
+    <div class="setting-value">
+      {{wizard-mapper
+        inputs=field.content
+        property="content"
+        onUpdate=(action "mappedFieldUpdated")
+        options=contentOptions}}
+    </div>
+  </div>
+{{/if}}
+
+{{#if showAdvanced}}
+  {{wizard-advanced-toggle showAdvanced=field.showAdvanced}}
+
+  {{#if field.showAdvanced}}
+    <div class="advanced-settings">
+      
+      {{#if isCategory}}
+        <div class="setting">
+          <div class="setting-label">
+            <label>{{i18n "admin.wizard.field.property"}}</label>
+          </div>
+          
+          <div class="setting-value">
+            {{combo-box
+              value=field.property
+              content=categoryPropertyTypes
+              onChange=(action (mut field.property))
+              options=(hash
+                none="admin.wizard.selector.placeholder.property"
+              )}}
+          </div>
+        </div>
+      {{/if}}
+      
+      <div class="setting">
+        <div class="setting-label">
+          <label>{{i18n "admin.wizard.translation"}}</label>
+        </div>
+        <div class="setting-value medium">
+          {{input
+            name="key"
+            value=field.key
+            class="medium"
+            placeholderKey="admin.wizard.translation_placeholder"}}
+        </div>
+      </div>
+      
+      {{#if validations}}
+        {{wizard-realtime-validations field=field validations=validations}}
+      {{/if}}
+    </div>
+  {{/if}}
+{{/if}}

```

Comment:
This diff looks weird. The reason being, I used the `sed` command to replace single quotes with double quotes but I've carefully looked at the file and there's nothing to worry.